app/models/scholarship_application.rb

```diff
@@ -0,0 +1,15 @@
+class ScholarshipApplication < ApplicationRecord
+  validate :user_is_verified
+
+  belongs_to :user
+  belongs_to :scholarship
+
+  delegate :name, to: :user
+  delegate :email, to: :user
+
+  def user_is_verified
+    unless self.user.verified
```

Comment:
Since this is an attribute on the `User` model this is a good candidate for delegation as well.

That would let us do `scholarship_application.user_verified`